spring-cloud-dataflow-core/src/main/java/org/springframework/cloud/dataflow/core/dsl/DestinationNode.java

```diff
@@ -56,7 +56,7 @@ public String stringify(boolean includePositionalInfo) {
 
 	@Override
 	public String toString() {
-		return getDestinationName();
+		return ":" + getDestinationName();
```

Comment:
Destination nodes have the `:` prefix, so if we don't have this, then as glenn mentions, we don't generate the correct DSL representation.  I agree, it is the type of change that can introduce regressions, but ATM our tests pass and can't think of anything, can you?